routes/index.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (express) {
   router.use('/user', require('./users')(express))
   router.use('/post', require('./posts')(express))
   router.use('/feed', require('./feed')(express))
+  router.use('/search', require('./search')(express))
```

Comment:
No. Search is a fundamentally different endpoint than `/api/user`. If, and that's a big *if* we implemented other resources, like `tags` or similar, than search could encompass that as well and still make sense as an endpoint. 

I'm not going to make another oven each time I want to bake a new pan of cookies.